src/Sylius/Bundle/WebBundle/Controller/Backend/FormController.php

```diff
@@ -45,8 +45,14 @@ public function showAction($type, $template)
      */
     public function filterAction($type, $template)
     {
+        $request = $this->get('request_stack')->getMasterRequest();
```

Comment:
No, because this controller is called inside a sub request and we need the master request.